### ISSUE TITLE
vikidia_fr_all_2017-02 no longer exist

### ISSUE THREAD

**Issue by letompouce** [ideascube/catalog-i-o#14]
([reported by thiolliere](https://github.com/ideascube/ansiblecube/issues/129))
in kiwix.yml catalog vikidia.fr link to http://download.kiwix.org/portable/vikidia/kiwix-0.9+vikidia_fr_all_2017-02.zip which is not found on the server

**Comment by letompouce** [comment]
Actual report from this morning cronjob:
```
/portable/wikipedia/kiwix-0.9+wikipedia_rn_all_2016-05.zip
/portable/wiktionary/kiwix-0.9+wiktionary_fr_all_2016-11.zip
/portable/wikisource/kiwix-0.9+wikisource_fr_all_2016-08.zip
/portable/vikidia/kiwix-0.9+vikidia_fr_all_2017-02.zip
/portable/wiktionary/kiwix-0.9+wiktionary_ku_all_2016-05.zip
/portable/wikipedia/kiwix-0.9+wikipedia_de_all_2017-01.zip
/portable/wikipedia/kiwix-0.9+wikipedia_ps_all_2016-05.zip
/portable/wikipedia/kiwix-0.9+wikipedia_sw_all_2016-05.zip
/portable/vikidia/kiwix-0.9+vikidia_es_all_2017-02.zip
/portable/other/kiwix-0.9+phet_en_all_2016-06.zip
/portable/vikidia/kiwix-0.9+vikidia_en_all_2017-02.zip
/portable/wikiquote/kiwix-0.9+wikiquote_fr_all_2016-11.zip
/portable/wikiquote/kiwix-0.9+wikiquote_fa_all_2016-11.zip
```
We usually let the 404 stacking together until we need one of those. Then we update them all at once. Which I will do today since you've been bitten!

**Comment by letompouce** [comment]
Fixed by c2ddca86aa597d97037cea95e2f6133a2ada0733

Thanks!

**Comment by letompouce** [comment]
Oh, for the record, [the cronjob runs on kawax](https://github.com/ideascube/kawax-scripts/blob/master/kawax#L82-L96).
